File: dotnet/cafdemalihapa/Applications/NotepadPlusPlusFileManagement/ProcessInformationGroup.cs
namespace cafdemalihapa.Applications.NotepadPlusPlusFileManagement
{
    public class ProcessInformationGroup
    {
        public List<ProcessInformation>? Group { get; set; } = [];

        public void Add(ProcessInformation processInformation)
        {
            Group?.Add(processInformation);
        }
        public void AddInFront(ProcessInformation processInformation)
        {
            Group?.Insert(0, processInformation);
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Applications/NotepadPlusPlusFileManagement/ProcessInformationGroup.fs
namespace cafdemwimu.console.Applications.NotepadPlusPlusFileManagement

open System.Collections.Generic

[<AllowNullLiteral>]
type ProcessInformationGroup() =
    member val Group: List<ProcessInformation> = List<ProcessInformation>() with get, set

    member this.Add(processInformation: ProcessInformation) =
        this.Group.Add(processInformation)

    member this.AddInFront(processInformation: ProcessInformation) =
        this.Group.Insert(0, processInformation)
